recipes/starshot/config/easy_breadcrumb.settings.yml

```diff
@@ -0,0 +1,41 @@
+applies_admin_routes: true
```

Comment:
Question -- is this config the same thing as ships with `easy_breadcrumb`? If so, it's not necessary. The module's simple config will be installed by default. If we're just trying to override one or two of its values, then that should be done by a config action.

Reply:
@phenaproxima Yeah, it's the config that ships with the module. It works well enough out of the box with the default configurations. It can be removed. Awaiting @thejimbirch response post which will update the PR.
Thanks